redis/src/aio.rs

```diff
@@ -598,6 +648,10 @@ where
     fn get_db(&self) -> i64 {
         self.db
     }
+
+    fn execute_push_message(&mut self, _kind: String, _data: Vec<Value>) {
+        // TODO - implement handling RESP3 push messages
```

Comment:
now that I think of it, maybe we want to use unimplemented, in order to be sure that users are aware?
`unimplemented!("RESP3 pubsub isn't implemented yet");`

what do you think?

Reply:
then it'll panic when any push message received 🤔

Reply:
right - that way people will be aware of the fact that pubsub isn't supported, instead of trying to understand why no message was received.
@jaymell, what's your opinion? silently ignoring, or loudly panicing?